datasets/medmcqa/README.md

```diff
@@ -0,0 +1,211 @@
+---
+
+annotations_creators:
+- no-annotation
+language_creators:
+- expert-generated
+languages:
+- en
+licenses:
+- apache-2.0
+multilinguality:
+- monolingual
+size_categories:
+- 100K<n<1M
+source_datasets:
+- original
+task_categories:
+- question-answering
```

Comment:
we defined a high level `multiple-choice` tag recently, see the [tagging app](https://huggingface.co/spaces/huggingface/datasets-tagging)
```suggestion
- question-answering
- multiple-choice
```